Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -29,6 +29,7 @@ struct FPTIBatchData: Codable {
     /// Encapsulates a single event by it's name and timestamp.
     struct Event: Codable {
 
+        let appSwitchURL: URL?
```

Comment:
I think here and in the init this would be a string. And when we send the event should instead be using `appSwitchURL.absoluteString`.